02-lsp_extension/02.00-lsp_intro.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p>Nice. The history really helps me understand the purpose of the tool</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nicole-brewer/jupyterlab-ide/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>